tests/unit/transformers/test_numerical.py

```diff
@@ -333,14 +328,12 @@ def test__fit_learn_rounding_scheme_true_inf(self):
         """Test ``_fit`` with ``learn_rounding_scheme`` set to ``True``.
 
         If the ``learn_rounding_scheme`` parameter is set to ``True``, and the data
-        contains values that are infinite, ``_fit`` should learn the ``_rounding_digits``
-        to be the min number of decimal places seen in the data with the infinite values
-        filtered out.
+        contains values that are infinite, ``_fit`` shouldn't round.
```

Comment:
This is misleading because it should round if there are infinite values so long as there are actual decimals. This test was just making sure that when we used to round to higher places, the infinite values were ignored.